### DISCUSSION TITLE
Command not working

### DISCUSSION THREAD

**Discussion by yoshiafk** [drizzle-mizzle/CharacterAI-Discord-Bot#89] [Q&A]
I have been adding the bot to discord, but the command won't work, is there another setting with the bot or something I should do?

**Comment by drizzle-mizzle** [comment]
What is the command that you're referring to, and what "won't work" means exactly?

**Reply by yoshiafk** [reply]
to be exact all commands as simple as set or find, and I mean by won't work, instead of doing what the command does, it gives another response

**Comment by yoshiafk** [comment]
I've been figure it out, there's something wrong with my prefix
